CalculatorWithTests.Test/CalculatorTests.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using Xunit;
+using CalculatorWithTests;
+
+namespace CalculatorWithTests.Tests
+{
+    public class CalculatorTests
+    {
+        [Theory]
+        [InlineData(1, 1, 2)]
+        [InlineData(-5, -2, -7)]
+        [InlineData(0, 0, 0)]
+        [InlineData(null, 5, 0)]
```

Comment:
`null` for an integer parameter?  If you want this ability, have a look at nullable types, such as `int?`, `bool?`, etc.